packages/dashboard-core-plugins/src/panels/ChartPanel.test.jsx

```diff
@@ -88,37 +98,56 @@ function makeChartPanelWrapper({
   );
 }
 
+function callUpdateFunction() {
+  MockChart.mock.calls[MockChart.mock.calls.length - 1][0].onUpdate();
+}
+
+function callErrorFunction() {
+  MockChart.mock.calls[MockChart.mock.calls.length - 1][0].onError();
+}
+
+function expectLoading() {
+  expect(screen.getAllByRole('img', { hidden: true }).length).toBe(2);
```

Comment:
While this does match, I don't think it captures what we _actually_ want to check here (that a loading spinner is shown). Consider following this up with another ticket to add an `aria-busy` tag to the loading spinner: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy#within_a_widget